spec/features/user_authentication_spec.rb

```diff
@@ -0,0 +1,84 @@
+require "rails_helper"
+
+describe "Authentication procedure" do
+  it "successful login\logout to portal" do
+    user = create(:user)
+    visit login_path
+    login_user(user.email, "1234")
+    expect(page).to have_content "Login successful"
+    click_link "Logout"
+    expect(page).to have_content "Logged out!"
+   end
```

Comment:
`end` at 11, 3 is not aligned with `it "successful login\logout to portal" do` at 4, 2